src/Polytopes/Polyhedron/properties.jl

```diff
@@ -87,7 +37,28 @@ julia> collect(F)
 """
 function faces(as::Type{T}, P::Polyhedron, face_dim::Int) where {T}
     if as == Polyhedron || as == Polyhedra
-        PolyhedronFacePolyhedronIterator(P,face_dim-size(lineality_space(P),1))
+        if (face_dim < 0)
+            return nothing
+        end
```

Comment:
More idiomatic would be
```suggestion
        face_dim >= 0 || return nothing
```